charmcraft.yaml

```
@@ -150,3 +150,9 @@ requires:
     optional: true
     description: |
       Integration to help users discover Mimir's deployment.
+config:
+  options:
+    max_global_exemplars_per_user:
+      default: 100000
+      description: The limit for exemplars
```

How about (please confirm):
```suggestion
      description: |
        The global limit for number of exemplars.
        When unset or set to a non-positive value, exemplar storage is disabled.
```